src/hotspot/share/opto/matcher.cpp

```diff
@@ -531,6 +538,9 @@ void Matcher::init_first_stack_mask() {
   *idealreg2spillmask[Op_RegD] = *idealreg2regmask[Op_RegD];
    idealreg2spillmask[Op_RegD]->OR(aligned_stack_mask);
 
+  *idealreg2spillmask[Op_RegVMask] = *idealreg2regmask[Op_RegVMask];
```

Comment:
I can still get a SEGV on arm (32bit) build. I think it's here?

\#
\# A fatal error has been detected by the Java Runtime Environment:
\#
\#  SIGSEGV (0xb) at pc=0xfee7ed8e, pid=18153, tid=18162
\#
\# JRE version: OpenJDK Runtime Environment (17.0) (fastdebug build 17-internal+0-git-85a6b6e69)
\# Java VM: OpenJDK Server VM (fastdebug 17-internal+0-git-85a6b6e69, mixed mode, serial gc, linux-arm)
\# Problematic frame:
\# V  [libjvm.so+0xaeed8e]  Matcher::init_first_stack_mask()+0x89d
\#
\# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P" (or dumping to //core.18153)
\#